docs/web/user_guide.md

```diff
@@ -616,9 +616,16 @@ filter arguments:
                         <minimum_bug_path_length>:<maximum_bug_path_length>.
                         Valid values are: "4:10", "4:", ":10"
   --tag [TAG [TAG ...]]
-                        Filter results by version tag names. This can be used
-                        only if basename or newname is a run name (on the
-                        remote server).
+                        Filter results by version tag names.
+  --open-reports-date TIMESTAMP
```

Comment:
I'm thinking about a bit different cli argument name:
```suggestion
  --open-reports-at TIMESTAMP
```
or
```suggestion
  --open-reports-at-date TIMESTAMP
```
What do you think @csordasmarton @bruntib @dkrupp ?

Reply:
@gyorb Now in the UI we use `Detection date` and `Fix date` for other date filters. To be consistent with these names it is better to use the original `--open-reports-date` filter option.